pom.xml

```diff
@@ -7,7 +7,7 @@
     <artifactId>openworld-java-sdk-core</artifactId>
     <name>EG Open World SDK :: Core</name>
     <description>Core Modules of EG Travel SDK</description>
-    <version>0.6.0</version>
+    <version>0.7.0</version>
```

Comment:
Can we use `0.7.1` please? As this is a little different from our published `0.7.0`.

Reply:
Sure